Add documentation for AIX and Intune parser (#663)
SEKOIA-IO/documentation: docs/xdr/features/collect/integrations/endpoint/microsoft_intune.md

uuid: c20528c1-621e-4959-83ba-652eca2e8ed0
name: Microsoft Intune
type: intake

## Overview

Microsoft Intune helps you protect your workforce's corporate data by managing devices and apps. Intune provides mobile device management (MDM) and mobile app management (MAM) from a secure cloud-based service that is administered using the Microsoft Endpoint Manager admin center. Using Intune, you ensure your workforce's corporate resources (data, devices, and apps) are correctly configured, accessed, and updated, meeting your company's compliance policies and requirements.

This setup guide describe how to forward events produced by `Microsoft Intune` to SEKOIA.IO.

> 

Reviewer: Recent change on the documentation, to associate rule to datasource, we should include a new autogenerated document
```suggestion
{!_shared_content/operations_center/detection/generated/suggested_rules_c20528c1-621e-4959-83ba-652eca2e8ed0_do_not_edit_manually.md!}

```

{!_shared_content/operations_center/integrations/generated/microsoft-intune_do_not_edit_manually.md!}

## Configure

### Prerequisites

To forward events to SEKOIA.IO, create [an Event hub namespace and an Event hub](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create) with a consumergroup.

Once created, in your EventHubs, go to `Setting > Shared access policies`.
Create a new policy with the option `Listen` then copy the `Connection string-primary key`.

Create a [Storage accounts](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create) or use an existing one. Go to `Data storage > containers` and create a new container.
Then go to `Security + networking > Access keys` and copy the key1 `Connection string`

> Configure Azure Front door to stream its logs to the EventHub with [this guide](https://docs.microsoft.com/en-us/azure/frontdoor/standard-premium/how-to-logs#configure-logs).

Reviewer: ```suggestion
Configure Microsoft Intune door to stream its logs to the EventHub with [this guide](https://learn.microsoft.com/en-us/mem/intune/fundamentals/review-logs-using-azure-monitor).
```

This setup guide describe how to forward events produced by `Microsoft Intune` to SEKOIA.IO.
{!_shared_content/operations_center/detection/generated/suggested_rules_c20528c1-621e-4959-83ba-652eca2e8ed0_do_not_edit_manually.md!}

{!_shared_content/operations_center/integrations/generated/microsoft-intune_do_not_edit_manually.md!}

## Configure

### Prerequisites

To forward events to SEKOIA.IO, create [an Event hub namespace and an Event hub](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create) with a consumergroup.

Once created, in your EventHubs, go to `Setting > Shared access policies`.
Create a new policy with the option `Listen` then copy the `Connection string-primary key`.

Create a [Storage accounts](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create) or use an existing one. Go to `Data storage > containers` and create a new container.

> Then go to `Security + networking > Access keys` and copy the key1 `Connection string`

Reviewer 2: ```suggestion
Then go to `Security + networking > Access keys` and copy the key1 `Connection string`.
```
